base/parse.jl

```diff
@@ -126,27 +130,49 @@ end
 
 function tryparse_internal(::Type{Bool}, sbuff::Union{String,SubString},
         startpos::Int, endpos::Int, base::Integer, raise::Bool)
-    len = endpos-startpos+1
-    p = pointer(sbuff)+startpos-1
+    null = Nullable{Bool}()
```

Comment:
I don't really see the point of defining this variable. Why not write `Nullable{Bool}()` where you need it?